### ISSUE TITLE
WaTTS 1.6.1: "plugin does not support stdin, it will be disabled"

### ISSUE THREAD

**Issue by marcocaberletti** [indigo-dc/tts#548]
Hi, 
I'm trying to deploy the latest version of WaTTS.
The watts service runs, but all services are disabled (I use `info` and `x509` services). 
Into log file, I found:
```
[warning] <0.850.0>@watts_init:log_warning:1020 Init: service <<"info">>: plugin does not support stdin, it will be disabled
```
Same warning for all services.

In the configuration file, I put the same options used deploying older versions (in particular 1.3.0)
```
service.info.description = Simple Info Service
service.info.credential_limit = 1
service.info.connection.type = local
service.info.cmd = /var/lib/watts-plugins/info.py
service.info.parallel_runner = infinite
service.info.authz.allow.any.sub.any = true
```
In version 1.3.1, this configuration runs without the warning pasted above.

What am I missing? Do I forget some option?

Thanks,
Marco

**Comment by bwegh** [comment]
Hi Marco,

you need to enable insecure plugins using the
allow_insecure_plugins setting.
documented here https://indigo-dc.gitbooks.io/token-translation-service/content/config_server.html

cheers,
Bas

**Comment by marcocaberletti** [comment]
Sorry, I didn't see that option.
I've fixed the configuration file, deployed 1.6.1 and now it works.

Thanks!

**Comment by bwegh** [comment]
btw: it was also documented in the "upgrade watts" part of the admin document: 
https://github.com/indigo-dc/tts/blob/master/gitbook/admin.md
